src/Libraries/CoreNodes/DSCoreNodes.Migrations.xml

```diff
@@ -22,11 +22,23 @@
   </priorNameHint>
   <priorNameHint>
     <oldName>DSCore.File.ExportToCSV</oldName>
-    <newName>DSCore.IO.File.ExportToCSV</newName>
+    <newName>DSOffice.Data.ExportCSV</newName>
```

Comment:
I personally think it's not very necessary as `DSCore.IO.File.ExportCSV` only exists since this PR from a few days ago: #7776